ccmlib/scylla_node.py

```diff
@@ -318,7 +318,7 @@ def _wait_java_up(self, data):
                 s.close()
             except:
                 pass
-            time.sleep(1)
+            time.sleep(0.1)
```

Comment:
Not only was 1 second sleep too much here (this wait is usually around 250ms on my laptop), this silly loop did an extra sleep after succeeding! The sleep should be where the pass is :-) @Orenef11 just re-discovered this problem and fixed it in PR https://github.com/scylladb/scylla-ccm/pull/356